content/blog/trustless-lightning-payments-for-content.md

```diff
@@ -0,0 +1,357 @@
+---
+
+title: "Trustless Lightning Payments for Content"
+subtitle: "How PTLC payment channels in Lightning enable decentralized social media."
+date: 2022-01-07T00:00:00+00:00
+authors:
+  - name: Jonathan Zernik
+    github: yzernik
+    twitter: yzernik
+tags:
+  - "Lightning"
+  - "PTLC"
+  - "Bitcoin"
```

Comment:
not sure if the Bitcoin tag is needed on bitcoin-dev blog :)

Reply:
Done.